src/components/MainScreen/index.tsx

```diff
@@ -1,10 +1,33 @@
-import { Flex } from '@chakra-ui/react';
+import { Box, Flex } from '@chakra-ui/react';
 import { SideBar } from '@/components/SideBar';
 import { SideDrawer } from '@/components/SideDrawer';
+import { ProductCategory } from '@/components/ProductCategory';
+
+export const MainScreen = () => {
+  const listOfCategoriesWithItems = [{
+    title: 'Fruit and Vegetables',
+    items: ['Avocado ', 'Banana', 'Bunch of purple carrots 5pcs', 'Watermelon', 'Mandarin Nadorcott', 'Orange', 'Spinach', 'Cucumber', 'Broccoli']
+  }, {
+    title: 'Beverages',
+    items: ['Soda ', 'Cola', 'Orange Juice', 'Beer Pilsener 6x', 'Wine']
+  },
+  {
+    title: 'Bakery',
+    items: ['Bread', 'Whole Grain Bread', 'Chocolate Cake', 'Butter Cake', 'Croissant', 'Pan au Chocolat']
+  }
+  ];
+
+  return (
+    <Flex bg="#fafafe" w="full" justifyContent="space-between" >
+      <SideBar />
+      <Box w='100%' h='100vh' overflowY='scroll'>
+        <ProductCategory {...listOfCategoriesWithItems[0]}></ProductCategory>
+        <ProductCategory {...listOfCategoriesWithItems[1]}></ProductCategory>
+        <ProductCategory {...listOfCategoriesWithItems[2]}></ProductCategory>
+      </Box>
+      <SideDrawer />
+    </Flex>
+  )
+};
+
```

Comment:
This fact is not relevant to us and what you did is not wrong, but it is interesting for me to talk about it.

Variables created inside a React component are recreated on every new render, a good practice for this is to move it outside the component, this would be relevant in a real life project.

And another tip is to add the `mock` prefix to the constant as it makes it clearer to use this constant for the next dev

```suggestion
import { ProductCategory } from '@/components/ProductCategory';

const mockListOfCategoriesWithItems = [{
    title: 'Fruit and Vegetables',
    items: ['Avocado ', 'Banana', 'Bunch of purple carrots 5pcs', 'Watermelon', 'Mandarin Nadorcott', 'Orange', 'Spinach', 'Cucumber', 'Broccoli']
  }, {
    title: 'Beverages',
    items: ['Soda ', 'Cola', 'Orange Juice', 'Beer Pilsener 6x', 'Wine']
  },
  {
    title: 'Bakery',
    items: ['Bread', 'Whole Grain Bread', 'Chocolate Cake', 'Butter Cake', 'Croissant', 'Pan au Chocolat']
}];

export const MainScreen = () => {
  return (
    <Flex bg="#fafafe" w="full" justifyContent="space-between" >
      <SideBar />
      <Box w='100%' h='100vh' overflowY='scroll'>
        <ProductCategory {...mockListOfCategoriesWithItems[0]}></ProductCategory>
        <ProductCategory {...mockListOfCategoriesWithItems[1]}></ProductCategory>
        <ProductCategory {...mockListOfCategoriesWithItems[2]}></ProductCategory>
      </Box>
      <SideDrawer />
    </Flex>
  )
};

```

Reply:
I don't know if it was **prettier** that formatted it like this, but the indentation seems a little strange to me, is your **prettier** working correctly? I said that because of line 17 and 18.